spek-ide-plugin/intellij-base/src/main/kotlin/org/spekframework/intellij/support/SpekCommonProgramRunConfigurationParameters.kt

```diff
@@ -0,0 +1,79 @@
+package org.spekframework.intellij.support
+
+import com.intellij.execution.CommonProgramRunConfigurationParameters
+import com.intellij.execution.configuration.EnvironmentVariablesComponent
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.JDOMExternalizerUtil
+import org.jdom.Element
+import org.spekframework.intellij.ProducerType
+import org.spekframework.spek2.runtime.scope.Path
+import org.spekframework.spek2.runtime.scope.PathBuilder
+
+abstract class SpekCommonProgramRunConfigurationParameters(private val _project: Project): CommonProgramRunConfigurationParameters {
```

Comment:
why the underscore?

Reply:
it conflicts with `CommonProgramRunConfigurationParameters` which has a `getProject()`.